### PR TITLE
Pallet updates according to latest discussions

### PR DESCRIPTION
**Pull request by vasylenko-yevhen**
- Additional validations when creating withdrawal proposal.
- Added missing docstrings.
- Updated tests.

### CODE REVIEW

File: control/src/lib.rs [hunk 1]
```diff
@@ -641,7 +657,7 @@ impl<T: Config> Pallet<T> {
 		}
 	}
 
-	fn remove(org_id: T::Hash, account: T::AccountId) -> DispatchResult {
+	fn remove_org_member(org_id: T::Hash, account: T::AccountId) -> DispatchResult {
```
Comment (vovacha) [inline control/src/lib.rs:660]: Let's finish our long discussion about function names right here.
I'd stick to "do_{extrinsic_name}" format.
Example FRAME Uniques: https://github.com/paritytech/substrate/blob/master/frame/uniques/src/functions.rs#L25

Or, please, suggest other options.

**Review by vovacha — COMMENTED** [review]
Looks good, the only thing - I'd like to close the topic about naming convention.

File: control/src/lib.rs (hunk 1 above)
Comment (vasylenko-yevhen) [reply control/src/lib.rs:660]: Let's use `do_...` naming.

**Comment by 2075** [comment]
👏

**Pull request merged**